Dockerfile

```diff
@@ -113,23 +114,18 @@ RUN chown -R nginx:nginx /var/lib/nginx/ && \
     touch /var/run/supervisor.sock && \
     chmod 777 /var/run/supervisor.sock
 
-COPY --from=builder --chown=nginx:nginx /usr/src/humhub /var/www/localhost/htdocs/
-COPY --chown=nginx:nginx humhub/ /var/www/localhost/htdocs/
+COPY --from=builder --chown=nginx:nginx /usr/src/humhub /tmp/humhub
+COPY --chown=nginx:nginx humhub /tmp/humhub
 
 RUN mkdir -p /usr/src/humhub/protected/config/ && \
-    cp -R /var/www/localhost/htdocs/protected/config/* /usr/src/humhub/protected/config/ && \
-    echo "v${HUMHUB_VERSION}" >  /usr/src/humhub/.version
```

Comment:
I moved the .version file to live with the complete source files (`/tmp/humhub` in my case). I moved it for two reasons: 1) It seems to me that the version number file should live with the files that are more specific to a version, rather than with the config files that are more likely to survive between version changes, and 2) the old `ENV HUMHUB_VERSION` variable didn't seem to be surviving the first `FROM` builder block, so I moved it into the first block. This was before I merged your changes, where you moved to an `ARG HUMHUB_VERSION` instead, so that may change the behavior - I haven't looked into it yet.